src/constants.ts

```diff
@@ -0,0 +1,756 @@
+export const CONTRACT_ADDRESS = '0x16703afb468e4ba88380c2a2fda1aa4c5ec7b264'
```

Comment:
since this file is used only for testing, let's move it to a `/tests` folder

Reply:
Okay, I'm coming!